Jenkinsfile_CNP

```diff
@@ -10,7 +10,8 @@ def secrets = [
     'wa-${env}' : [
     ],
     's2s-${env}': [
-        secret('microservicekey-wa-case-event-handler', 'S2S_SECRET_CASE_EVENT_HANDLER')
+        secret('microservicekey-wa-case-event-handler', 'S2S_SECRET_CASE_EVENT_HANDLER'),
+        secret('microservicekey-wa-workflow-api', 'S2S_SECRET_WORKFLOW_API'),
```

Comment:
Do we need the workflow api secret

Reply:
TODO: raise PR with rpe

Reply:
https://github.com/hmcts/camunda-bpm/pull/84